packages/frontend/src/components/common/Table/ScrollableTable/TableBody.tsx

```diff
@@ -212,7 +212,15 @@ const TableRow: FC<TableRowProps> = ({
         </Tr>
     );
 };
-
+const getMaxRowHeight = (row: Row<ResultRow>) => {
+    return row.getVisibleCells().reduce((acc, cell) => {
+        const lines = (
+            cell.getValue() as ResultRow[0] | undefined
+        )?.value?.formatted?.split('\n');
```

Comment:
this only takes into account new lines (which wsa the issue the customer reported)
This will not take into account long "line-break" text (we'll have to calculate the width of the cell, I believe this is not in scope)

Reply:
added a comment in code